LayoutTests/pointer-lock/pointer-lock-option-unadjusted-movement.html

```diff
@@ -0,0 +1,79 @@
+<!DOCTYPE HTML>
+<html>
+<head>
+<script src="../http/tests/resources/js-test-pre.js"></script>
+<script src="../http/tests/resources/pointer-lock/pointer-lock-test-harness.js"></script>
+</head>
+<body>
+<div>
+  <div id="target1"></div>
+</div>
+<script>
+    description("Test pointer lock unadjustedMovement option.")
```

Comment:
Could this be a web platform test? Are there web platform tests for this feature?

Reply:
There aren't _yet_. The existing web platform tests for pointerlock in WebKit match what's in https://github.com/web-platform-tests/wpt/tree/master/pointerlock, and I did verify that they all still pass with my code changes.

I think the order of operations could go something like this:

1. This PR gets merged and PointerLockOptions becomes an experimental feature in WebKit.
2. https://github.com/w3c/pointerlock/pull/49/ gets merged. The reason it hasn't been is because it only has an implementation in Chromium and nowhere else so far. I'm trying to solve that with this PR and unwedge it.
3. Either I or someone from Chromium can propose a new web platform test that tests the unadjustedMovement option as well as the Promise return for requestPointerLock.
4. Import the new WPT to WebKit and verify it passes.
5. Mark PointerLockOptions as stable in WebKit.

Reply:
I agree that this should be a web platform test. Having a web platform test is also a prerequisite for having that PR merged (speaking as Chair of the WebApps WG).

Reply:
(not that it doesn't need to be sent to WPT first... you can add it here, and the we can upstream it to WPT)

Reply:
Let's aim to just make this a WPT that can be upstreamed later?

Reply:
When you say make it a WPT, do you mean leave it in LayoutTests/pointer-lock directory but write it against testharness.js instead of js-test-pre.js/pointer-lock-test-harness.js?

Reply:
I imagined (a) writing against testharness.js and (b) placing the test under `LayoutTests/http/wpt/pointer-lock/`

Reply:
OK, I added 3 new tests under LayoutTests/http/pointerlock.

@aprotyas @marcoscaceres please take a look and see if these match what you had in mind.

Reply:
Sorry, they should actually go into LayoutTests/imported/w3c/web-platform-tests/pointerlock/

Reply:
(oops, my apologies for the wrong info)